xoa/resource_xenorchestra_vm.go

```diff
@@ -1184,20 +1183,19 @@ type guestNetwork map[string][]string
 // of maps where each element represents a network interface.
 // Each map will contain the following keys: ip, ipv4 and ipv6. The values
 // will be a slice of ip addresses.
-// []map[string][]string{
-//   {
-//     "ip":   []string{"interface 1's IPs",
-//     "ipv4": []string{"interface 1's IPs",
-//     "ipv6": []string{"ip1", "ip2"}
-//   },
-//   {
-//     "ip":   []string{"interface 2's IPs",
-//     "ipv4": []string{"interface 2's IPs",
-//     "ipv6": []string{"ip1", "ip2"}
-//   },
-// }
-//
 //
+//	[]map[string][]string{
+//	  {
+//	    "ip":   []string{"interface 1's IPs",
+//	    "ipv4": []string{"interface 1's IPs",
+//	    "ipv6": []string{"ip1", "ip2"}
+//	  },
+//	  {
+//	    "ip":   []string{"interface 2's IPs",
+//	    "ipv4": []string{"interface 2's IPs",
+//	    "ipv6": []string{"ip1", "ip2"}
+//	  },
+//	}
```

Comment:
Was this change intentional? Unless this is a gofmt or other formatting tool change, I think it would be better to leave this as is.

Reply:
This is a formatting change by gofmt. Can be left out or stay in depending on your preference.

Reply:
Sounds good. I'm fine with it applying then otherwise we will continue to undo it's changes.